test/test-filter-rewriter.rb

```diff
@@ -137,4 +137,31 @@ def test_rewrite_multiple
                    rewrite("column1 @ \"value1\" && column2 @~ \"^(?!.*value2 | value3 | value4).+$\""))
     end
   end
+
+  class AndNotOperatorTest < self
+    def rewrite(filter, enabled = true)
+      super(filter,
+            :rewrite_and_not_operator => enabled)
+    end
+
+    def test_rewrite_one
+      assert_equal("(column1 @ \"value1\") &! (column2 @ \"value2\")",
+                   rewrite("(column1 @ \"value1\") && ! (column2 @ \"value2\")"))
+    end
+
+    def test_reference
```

Comment:
Is this test needed?
This implementation doesn't care any column name.

Reply:
Ah...
This is needless.